cmd/mqtt2prometheus.go

```diff
@@ -182,3 +189,27 @@ func setupExtractor(cfg config.Config) (metrics.Extractor, error) {
 	}
 	return nil, fmt.Errorf("no extractor configured")
 }
+
+func newTlsConfig(cfg config.Config) *tls.Config {
+	certpool := x509.NewCertPool()
+	pemCerts, err := ioutil.ReadFile(cfg.MQTT.CACert)
+	if err == nil {
```

Comment:
Could you please return an error here? Silently ignoring the error will lead to lot of user frustration while debugging the own setup.

Reply:
As for here, CA cert may not be required.
In that case, you can catch it in a later process.

Reply:
Yes but the user might specified an invalid path. E.g. a typo. In that case you will just silently ignore the error and the user wonders why the TLS configuration does not work.

Reply:
You're right. I've revised to return error.
If it does not exists CACert settings, skip here.